### ISSUE TITLE
Provide easier way to find logs of imagebuild

### ISSUE THREAD

**Issue by mustard-mh** [gitpod-io/gitpod#12500]
### Objective

Now we need to exec SQL below to get the correct imagebuild id, so that we can search it in logs. 

It would be great to have an easier way to reduce the steps we need to do

```
SELECT imageBuildInfo->>'$.log.url' FROM d_b_workspace_instance WHERE id='<instance_id>';
```

**Comment by geropl** [comment]
@mustard-mh Absolutely, yes! But so far webapp has no connection whatsoever to that abstraction.

:cloud: One nice way to move forward here would be to push https://github.com/gitpod-io/gitpod/issues/7845 over the finish line. As a side effect it would become trivial for us (WebApp) to consume updates for workspace type `imagebuild` and treat them just like any other workspace type. :lotus_position: :upside_down_face: 

related: https://github.com/gitpod-io/gitpod/issues/12474
